docker-compose.yml

```diff
@@ -55,7 +55,7 @@ services:
       - ./psalm.xml:/usr/src/code/psalm.xml
       - ./tests:/usr/src/code/tests
       - ./app:/usr/src/code/app
-      # - ./vendor:/usr/src/code/vendor
+      - ./vendor:/usr/src/code/vendor
```

Comment:
Might be better to comment this by default, it would make new contributors life harder as they will have to install all the dependencies with composer locally, especially if they don't use PHP often.

Reply:
I think this line also makes the tests to fail, makes sure to comment all vendor mounts (travis don't install dependencies locally - which is OK because otherwise the build will take longer time to run)

Reply:
Okay commenting it 👍